shared-libs/transitions/src/transitions/update_clinics.js

```diff
@@ -100,10 +100,16 @@ module.exports = {
 
       const form = change.doc.form && utils.getForm(change.doc.form);
       if (!form || !form.public_form) {
+        if (form && !form.public_form) {
```

Comment:
I think we can shorted this to:
```suggestion
        if (form) {
```

Reply:
Done.